dvjsharma/Drawn2Shoe
# Added lint and type checking issue #98

## Description

Added manual workflow run for lint and type checking

### Changes

Added nodejs.yml file in .github -> workflows 

## How to test

Go to github -> repo_name -> actions

Under actions select the workflow you want to run 
Then click on run workflow on the right side of the panel , it some errors occur , you can check it by viewing the summary


-- > Feel free to contact for any support or in case of any doubt
